test/e2e-image/namespace-overlays/topology/values.yaml

```diff
@@ -19,12 +20,7 @@ controller:
     periodSeconds: 1
   service:
     type: NodePort
-  electionID: ingress-controller-leader
-  ingressClassResource:
```

Comment:
any reason to remove this? IIRC on tests this may generate some conflict as multiple ingresses may try to reconcile it :)

Reply:
Ok I see you remove the charts after it. Maybe we should always remove charts and add this as part of AfterEach :)

Reply:
> any reason to remove this?

as this chart is expected to run alone (Serial), there is no need to disable the non-namespaced objects

> Maybe we should always remove charts and add this as part of AfterEach :)

TBH I was thinking about it but finally forgo because that would slightly increase the execution time of the general case where the removal of the ns is enough.
But I'm still leaning towards this idea as this would help to not fall into the trap when the removal of the chart deployment is necessary. Let me know what you think.

Reply:
I just read your other comment where you approve the idea ;)